src/index.js

```diff
@@ -0,0 +1,28 @@
+import dotenv from "dotenv";
+import cors from "cors";
+import express from "express";
+import { ApolloServer } from "apollo-server-express";
+
+import schema from "./schema";
+import resolvers from "./resolvers";
+import models from "./models";
+
+dotenv.config();
+const app = express();
+app.use(cors());
+
+
+const server = new ApolloServer({
+  typeDefs: schema,
+  resolvers,
+  context: {
+    models,
+    me: models.users[1],
+  },
+});
+
+server.applyMiddleware({ app, path: "/graphql" });
```

Comment:
Add the path value in the ENV file and call here from the ENV.

Reply:
Whenever i call it from the `.env`, it brings an error `Cannot POST /graphql`. My `.env` file looks like this:
`PATH='/graphql'
PORT=8000`. Kindly help me here... @verdotte 
Note that the port path is working but the only problem is on `process.env.PATH`